Bridges/HttpKernel.php

```diff
@@ -238,14 +239,33 @@ protected function mapResponse(HttpResponse $reactResponse, SymfonyResponse $syR
             $headers['Set-Cookie'] = $cookies;
         }
 
-        $reactResponse->writeHead($syResponse->getStatusCode(), $headers);
+        if ($syResponse instanceof SymfonyStreamedResponse) {
+            $reactResponse->writeHead($syResponse->getStatusCode(), $headers);
+
+            // asynchronously get content
+            ob_start(function($buffer) use ($reactResponse) {
+                $reactResponse->write($buffer);
+                return '';
+            }, 4096);
+
+            $syResponse->sendContent();
 
-        $stdOut = '';
-        while ($buffer = @ob_get_clean()) {
-            $stdOut .= $buffer;
+            // flush remaining content
+            @ob_end_flush();
+            $reactResponse->end();
         }
+        else {
+            ob_start();
+            $content = $syResponse->getContent();
```

Comment:
@marcj I don't but this is actually already the case in the current implementation with buffering started here https://github.com/php-pm/php-pm-httpkernel/blob/master/Bridges/HttpKernel.php#L84

Is there a specific function you had intended to buffer in the current implementation? Then I could limit it to that one.